### PR TITLE
Refactor/added to favorites user feedback

### PR DESCRIPTION
**Pull request by jmyacobn**
Summary of the work done:
- Remove "Save Recipe" button after user adds recipe
- Display message above recipe image letting the user know their recipe was saved
- Add padding to top of recipe image for design aesthetics (enhancement from Part 1)
- Change color of the title to brown for great contrast (accessibility concern)

Contributed to code:

- Jenn

Needs to review:

- Kiko and Lauren

Additional notes about any issues/questions remaining around this code:

- Kiko, I quickly added the padding we talked about to the single recipe page!
- Also, while I was in the CSS, I went ahead and changed the title color that was flagged when we checked the accessibility.

### CODE REVIEW

**Review by knpausch — CHANGES_REQUESTED** [review]
Great first fix, however I noticed when I save a recipe > go home > click that same recipe, that the 'save button' is still there. Or if you click on the favorite recipe from your favorites, it still shows the 'save button'. You may want to add the hide feature on all other pages (on recipe detail page after favorites view, after home view) or create a global variable (let hasBeenSaved = true) and use that as a conditional to check on each page

**Comment by jmyacobn** [comment]
@knpausch, I made the changes and pushed again. Do you mind checking the functionality for approval! Thank you for catching that bug!!

File: src/scripts.js [hunk 1]
```diff
@@ -127,6 +127,10 @@ function displayRecipeDetailPage(event) {
     displayRecipeInstructions(event)
     displayRecipeTotalCost(event)
     displayRecipeIngredients(event)
+    if(user.recipesToCook.includes(foundRecipe)) {
+      hide(favoriteRecipeButton)
+      recipe.insertAdjacentHTML("afterBegin", `<p class=recipe-message>This recipe has been added to favorites!</p>`)
```
Comment (FrazierLE) [inline src/scripts.js:132]: So cool how you figured this out!

**Review by FrazierLE — APPROVED** [review]

**Comment by knpausch** [comment]
Once I pulled the newest updates from origin Refactor/added to favorites user feedback, I was able to run the code and the save button is hidden at appropriate times. Ready to merge

**Pull request merged**